Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -5290,7 +5290,7 @@ def build_logs_query(address_src: Optional[str], address_dst: Optional[str], ip_
 @logger
 def panorama_query_logs(log_type: str, number_of_logs: str, query: str, address_src: str, address_dst: str, ip_: str,
                         zone_src: str, zone_dst: str, time_generated: str, time_generated_after: str, action: str,
-                        port_dst: str, rule: str, url: str, filedigest: str):
+                        port_dst: str, rule: str, url: str, filedigest: str, show_detail: str):
```

Comment:
```suggestion
                        port_dst: str, rule: str, url: str, filedigest: str, show_detail: str = False):
```

Lets make sure we do not affect other commands that might be using this function.